step.yml

```diff
@@ -6,124 +6,125 @@ website: https://github.com/bitrise-steplib/steps-export-xcarchive
 source_code_url: https://github.com/bitrise-steplib/steps-export-xcarchive
 support_url: https://github.com/bitrise-steplib/steps-export-xcarchive/issues
 host_os_tags:
-  - osx-10.10
+- osx-10.10
 project_type_tags:
-  - ios
-  - cordova
-  - ionic
-  - react-native
-  - flutter
-  - xamarin
+- ios
+- cordova
+- ionic
+- react-native
+- flutter
+- xamarin
 type_tags:
-  - utility
+- utility
 is_always_run: false
 is_skippable: false
 toolkit:
   go:
     package_name: github.com/bitrise-steplib/steps-export-xcarchive
 inputs:
-  - archive_path: $BITRISE_XCARCHIVE_PATH
-    opts:
-      title: iOS or tvOS archive path
-      summary: iOS or tvOS archive path
-      description: |-
-        Path to the iOS or tvOS archive (.xcarchive) which should be exported.
-      is_required: true
-  - export_method: "auto-detect"
-    opts:
-      title: "Select method for export"
-      summary: ""
-      description: |-
-        `auto-detect` option is **DEPRECATED** - use direct export methods!
+- archive_path: $BITRISE_XCARCHIVE_PATH
+  opts:
+    title: Archive path
+    summary: Specifies the archive that should be exported.
+    description: |-
+      Specifies the archive that should be exported.
 
-        Describes how Xcode should export the archive.   
-        
-        If you select `auto-detect`, the step will figure out the proper export method  
-        based on the provisioning profile embedded into the generated xcode archive.
-      value_options:
-      - "auto-detect"
-      - "app-store"
-      - "ad-hoc"
-      - "enterprise"
-      - "development"
-      is_required: true
-  - upload_bitcode: "yes"
-    opts:
-      title: "Include bitcode"
-      summary: ""
-      description: |-
-        For __App Store__ exports, should the package include bitcode?
-      value_options:
-      - "yes"
-      - "no"
-      is_required: true
-  - compile_bitcode: "yes"
-    opts:
-      title: "Rebuild from bitcode"
-      summary: ""
-      description: |-
-        For __non-App Store__ exports, should Xcode re-compile the app from bitcode?
-      value_options:
-      - "yes"
-      - "no"
-      is_required: true
-  - team_id: 
-    opts:
-      title: "The Developer Portal team to use for this export"
-      summary: ""
-      description: |-
-        The Developer Portal team to use for this export.
+    The input value sets `xcodebuild`’s `-archivePath` option.
+    is_required: true
 
-        Format example:
+- product: "app"
+  opts:
+    title: "Select a product to distribute"
+    summary: ""
```

Comment:
we came to the conclusion that if summary = description, we define a summary only and do not even list the description, for example: https://github.com/bitrise-steplib/steps-xcode-test/blob/master/step.yml#L251-L259